static-assets/components/cstudio-forms/controls/node-selector.js

```diff
@@ -355,17 +355,19 @@ YAHOO.extend(CStudioForms.Controls.NodeSelector, CStudioForms.CStudioFormField,
       $(itemEl).append(`<span class="name">${item.value}</span>`);
       if (item.include) {
         $(itemEl).append(`<span class="path">${item.include}</span>`);
-      } else {
+      } else if (item.inline) {
```

Comment:
Certain that this comes as a boolean?

Reply:
It's not, but only the embedded components have the property. Though I'll better validate the string in case the prop evolves